From 019735a9baa2666bd8b50d7ff98363e243fa613f Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Fri, 24 Jul 2020 11:12:56 +0200
Subject: [PATCH] imx8mm-var-dart: Enable I2C in console

Activate driver model for drivers to  have i2c commands
working in u-boot cmdline on the iMX8M Mini VAR-DART.

Works with VAR-DT8MCustomboard v1.4.

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 configs/imx8mm_var_dart_defconfig | 1 +
 1 file changed, 1 insertions(+)

diff --git a/configs/imx8mm_var_dart_defconfig b/configs/imx8mm_var_dart_defconfig
index 7d2b6c6107..32a53cef43 100644
--- a/configs/imx8mm_var_dart_defconfig
+++ b/configs/imx8mm_var_dart_defconfig
@@ -67,3 +67,4 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_FAT=y
 CONFIG_CMD_EXPORTENV=y
 CONFIG_CMD_BOOTSWITCH_POS=y
+CONFIG_DM_I2C=y
-- 
2.17.1

